content/en/providers/getting-started/operations_start_guide/_index.md

```diff
@@ -0,0 +1,208 @@
+[Security policies](https://wiki.egi.eu/wiki/EGI_CSIRT:Policies). She/he is also
+the primary contact for the NGI Security officer and EGI CSIRT. The Site
+Security Officer deals with security incidents and shall respond to enquiries in
+a timely fashion as defined in the collection of
+[security procedures and policies](https://wiki.egi.eu/wiki/EGI_CSIRT:Policies).
+
+### Regional level
+
+#### Regional Operator on Duty (ROD)
+
+A team responsible for solving problems/incidents in the infrastructure
+according to agreed procedures. ROD (teams) monitor the sites in their region,
+react to problems identified by the monitoring tools, and oversee problems
+through to their resolution. They ensure that problems are properly recorded and
+that the solutions progress according to specified time lines. They also provide
+support to sites and VOs as needed and provide informational flow to oversight
+bodies in cases of non-responsive sites. They ensure that all necessary
+information is available to all parties. The team is provided by each NGI and
+requires procedural knowledge on the process (rather than technical skills) for
+their work. New ROD team members are required to read the
+[ROD Welcome page](https://wiki.egi.eu/wiki/Regional_Operator_on_Duty_welcome)
+and be familiar with
+[ROD wiki page](https://wiki.egi.eu/wiki/Regional_Operator_on_Duty).
+
+#### NGI Security officer
+
+The member of EGI-CSIRT IRTF (Incident Response Task Force) currently on shift.
+Further information can be found at the
+[CSIRT:IRTF](https://wiki.egi.eu/wiki/EGI_CSIRT:IRTF) page. The role of the IRTF
+team is to handle day to day operational security issues and coordinate
+_Computer Security Incident Response_ across the EGI infrastructure. NGIs and
+Sites **MUST** respond in a timely manner to its requests and alerts.
+
+#### NGI Operations manager
+
+NGI operations manager is the contact point for all operational matters and
+represents the NGI within the
+[Operations Management Board](https://confluence.egi.eu/display/EGIBG/Operations+Management+Board).
+
+S/he is mainly responsible for:
+
+- keeping the NGI entry in the GOCDB up to date and for managing the status of
+  all sites under that NGI, and ensuring that that information is also kept
+  current
+- addressing problems with Site availability or reliability. The reports are
+  issued on a monthly basis and the NGI operations manager has 10 days to
+  respond to identified problems
+- attending regular
+  [Operations-Management-Board (OMB) meetings](https://indico.egi.eu/category/19/)
+
+All NGI operations management responsibilities are listed in the
+[RP OLA document](https://documents.egi.eu/document/463).
+
+### Project level
+
+#### Chief Operations Officer
+
+Chief Operations Officer leads EGI Operations, and is responsible for
+coordinating the operations of the infrastructure across the project.
+
+#### EGI CSIRT
+
+[EGI CSIRT](https://wiki.egi.eu/wiki/EGI_CSIRT:Main_Page) is an official
```

Comment:
Spell out acronyms on the first appearance on the page.

Reply:
Done.